pkgs/development/python-modules/imutils/default.nix

```diff
@@ -2,8 +2,14 @@
 , buildPythonPackage
 , fetchPypi
 , opencv3
+, opencv4
+, withOpenCV3 ? false
+, withOpenCV4 ? true
```

Comment:
I think we should choose one or other.

If there is a compelling use case, we should just have one flag to toggle to the non-default case. And just have the default case use opencv4